ui-tests/src/test/resources/features/connections/oauth.feature

```diff
@@ -25,6 +25,7 @@ Feature: Connections - OAuth
       | Google Calendar | Test-Google-Calendar-connection |
       | Twitter         | Test-Twitter-connection         |
 
+#    works OK on local computer, seems to be some unstability issue with jenkins job.
```

Comment:
seems like all the tests passed, is this remark really true?

Reply:
removed